### PR TITLE
support S02w3 follow up for both camps

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: export_kakuma_advert_contacts.py [hunk 1]
```diff
@@ -84,6 +84,9 @@
             if msg['uid'] in all_uuids or msg["consent_withdrawn"] == Codes.TRUE:
                 continue
 
+            # if msg["rqa_s02e03_NA"] == Codes.MATRIX_1:
```
Comment (as2388) [inline export_kakuma_advert_contacts.py]: I think you can just remove this

**Review by as2388 — APPROVED** [review]
I did some of this Kakuma-side on another branch, I trust you're using the same scheme id as there

File: export_kakuma_advert_contacts.py (hunk 1 above)
Comment (IsaackMwenda) [reply export_kakuma_advert_contacts.py]: updated thanks

**Comment by IsaackMwenda** [comment]
> I did some of this Kakuma-side on another branch, I trust you're using the same scheme id as there

Yes, I updated the same branch that you had worked on.

**Pull request merged**